### PR TITLE
[WIP]Slime Changes

### PR DESCRIPTION
**Pull request by as334**
These are still a work in progress, but I figured I might as well put the PR up.
Changes most of the slime reactions that produce chemicals to use Slime Jelly.
Adds a new class of chemicals that are synthesized by injecting slime cores with Slime Jelly.
These new chemicals interact, and become more powerful based off the presence of the other chemicals.
TODO:
- [ ] Finish moving over the rest of the slime related chems to the new file.
- [ ] Make all slime based reagents work off slime jelly.
- [ ] Give almost every slime a slime jelly reaction
- [ ] Add more interactions between the slime chems.
- [ ] Bug testing.
- [x] Further grudgecode Kor's codebabbi.

### CODE REVIEW

File: code/modules/reagents/Chemistry-Reagents/Slime-Reagents.dm [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+/datum/reagent/toxin/slime/medicine/on_mob_life(var/mob/living/M as mob)
+	if (prob(slimepower*20))
+		M.adjustBruteLoss(-slimepower/2)
+		M.adjustFireLoss(-slimepower/2)
+	else if(prob(40))
+		M << "You feel squishy"
+	..()
+/datum/reagent/toxin/slime/medicine/overdose_process(var/mob/living/M as mob)
+	if(prob(slimepower*25))
+		M.adjustBruteLoss(-slimepower)
```
Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Reagents/Slime-Reagents.dm]: this heals because of the negative

is this intentional?

Comment (as334) [inline code/modules/reagents/Chemistry-Reagents/Slime-Reagents.dm]: Whoops. No it was not.

**Comment by Incoming5643** [comment]
Keep in mind that if you're coding slime jelly reactions that slime people have slime jelly for blood. So if it's supposed to be a big gate to access, it won't really be.

**Comment by as334** [comment]
It's not really supposed to be much of a gate no. These are moreso late game xenobio thing, but if you have slime people then you're already at the point where you can produce it en mass easily already.

**Comment by WJohn** [comment]
C-could you also make it so slime attacks and weaknesses are based on their types?

Only yellow slimes would shock you, red ones set you on fire and so on? Burn the blue slimes as an alternative to water?

**Comment by as334** [comment]
That's beyond what i plan to do in this PR, but it's something I'd like to do. It'd basically require splitting off every type of slime into mob/living/simple_animal/slime/red or whatever, but it could be done.

**Comment by Kromgar** [comment]
So does this mean that green slimes require you use slime jelly? Because I don't want to have to overcomplicate just getting some god damn unstable mutagen

**Comment by KorPhaeron** [comment]
@WJohn 

Requiring fire for breeding/harvesting them would be a complete pain in the ass unless you want to give scientists atmos hardsuits

**Comment by KorPhaeron** [comment]
Also the stuff looks cool in general, but not a huge fan of gating mutation toxin/advanced mutation toxin behind purple slimes. I'd prefer each slime to have its own self contained reward/would prefer you add new reactions that use jelly instead of changing old ones.

**Comment by Lumipharon** [comment]
Bring back the bluespace slime AOE teleporter reaction

**Comment by Kromgar** [comment]
Don't gate green slimes behind slime jelly pls no

**Comment by Cheridan** [comment]
Progress on this seems to have halted, contact a maintainer if you want to revive it

**Pull request closed without merging**